### PR TITLE
Fastdfs

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

File: var/spack/repos/builtin/packages/fastdfs/fastdfs/package.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fastdfs(Package):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/fastdfs/fastdfs/package.py]: This file looks correct but is in the wrong directory.

File: var/spack/repos/builtin/packages/fastdfs/package.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fastdfs(Package):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/fastdfs/package.py]: This file is in the right directory but the install method needs to be updated.

File: var/spack/repos/builtin/packages/fastdfs/fastdfs/package.py (hunk 1 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/fastdfs/fastdfs/package.py]: Sorry, I made this mistake again...
I fixed it in https://github.com/spack/spack/pull/14378/commits/cfdacd180dc06b4bf792b02ef3664c181d0c0a00

File: var/spack/repos/builtin/packages/fastdfs/package.py (hunk 2 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/fastdfs/package.py]: I fixed it in cfdacd1

**Pull request merged**